project-novica/demo/src/main/resources/templates/home.html

```diff
@@ -60,5 +60,32 @@ <h4 class="modal-title">Add Question Form</h4>
         </div>
     </div>
 </div>
+<table class="table">
+    <thead class="thead-dark">
+    <tr>
+        <th scope="col">Title</th>
+        <th scope="col">Question</th>
+        <th scope="col">Creator</th>
+        <th scope="col">Date Created</th>
+        <th scope="col">Answer</th>
+        <th scope="col">Delete</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr th:each="q: ${questions}">
+        <td th:text="${q.title}">Title</td>
+        <td th:text="${q.question}">Question</td>
+        <td th:text="${q.creator}">Creator</td>
+        <td th:text="${q.createdAt}">Date</td>
+        <td>
+            <button onclick="window.location.href='question/answer';">View Answer
```

Comment:
Put question ID in the url as well, so you will be able to see all answers for a particular Questions.